fpdf/svg.py

```diff
@@ -257,9 +257,27 @@ def optional(value, converter=lambda noop: noop):
 }
 
 
+@force_nodocument
+def parse_style(svg_element):
+    """Parse `style="..."` making it's key-value pairs element's attributes"""
+    try:
+        style = svg_element.attrib["style"]
+    except KeyError:
+        pass
+    else:
+        for element in style.split(";"):
+            if not element:
+                continue
+
+            attr, value = element.split(":")
```

Comment:
Maybe we should guard the code againt badly formatted `style`?
Like `style="a:"` or `style=":b"`

Reply:
Sure! absolutely right. I'm not familiar with pull requests, is it common to put the code here or is it better to just do another commit?
I changed the code with this, now it should be more robust:
```
  pair = element.split(":")
  if len(pair) == 2 and pair[0] and pair[1]:
      attr, value = pair

      svg_element.attrib[attr.strip()] = value.strip() 
```

Reply:
> is it common to put the code here or is it better to just do another commit?

I'm not sure to correctly understand your question,
but usually people add commits as part of the same PR to address code reviews feedback.